rootfs/api/models/app.py

```diff
@@ -240,6 +263,7 @@ def delete(self, *args, **kwargs):
 
             try:
                 self._scheduler.ns.delete(self.id)
+                self._scheduler.ingress.delete(self.id, self.id)
```

Comment:
Since the line above deletes the application's namespace, is this line redundant?